test/functional/feature_shutdown.py

```diff
@@ -20,7 +22,10 @@ def set_test_params(self):
 
     def run_test(self):
         node = get_rpc_proxy(self.nodes[0].url, 1, timeout=600, coveragedir=self.nodes[0].coverage_dir)
+        # force connection establishment
+        node.getblockcount()
         Thread(target=test_long_call, args=(node,)).start()
+        sleep(1)
```

Comment:
I don't like hardcoding a second of sleep here, hardly seems robust.

Reply:
Agree, this should be replaced by something like:
```py
wait_until(lambda: self.getrpcinfo()['active_workers'] > 1)
```
Other suggestions?